ignite/base/events_driven.py

```diff
@@ -114,6 +116,8 @@ def add_event_handler(self, event_name: Any, handler: Callable, *args: Any, **kw
         Returns:
             :class:`~ignite.base.events.RemovableEventHandle`, which can be used to remove the handler.
         """
+        from ignite.engine.events import Events
+
```

Comment:
Put this import near the usage, line 134 : `event_args = (Exception(),) if event_name == Events.EXCEPTION_RAISED else ()`
and leave a comment saying that we have to redesign this part of code.